clear/main.py

```diff
@@ -545,6 +546,27 @@ def route(prefix, vrf, namespace):
         command += ['-n', str(namespace)]
     clicommon.run_command(command)
 
+@click.option('--all', '-a', is_flag=True, help='Delete also compressed logs')
+@cli.command()
+def logging(all):
+    """Clear logging files"""
+    if os.path.exists("/var/log.tmpfs"):
```

Comment:
Do we not plan to delete existing logs in `/var/log/` if both the directories `/var/log/` and `/var/log.tmpfs/` have syslogs?

Reply:
Yes, thanks.